.github/composite-actions/install-composer-deps/action.yml

```diff
@@ -8,6 +8,41 @@ inputs:
 runs:
   using: "composite"
   steps:
+    # Warning!
+    # Use $GITUB_ENV instead of $GITHUB_OUTPUT.
```

Comment:
```suggestion
    # Use $GITHUB_ENV instead of $GITHUB_OUTPUT.
```

Reply:
OK, i scanned whole and fixed more typos, thanks for pointing